File: src/app/Entity/Userreg.ts
export class Userreg{
    userId:number;
	userfName:string;	
	userlName:string;
	username:string;
	userPassword:string;
	userEmail:string;
	userContactNo:string;
	userDOB:Date;

    constructor(userId:number, userfName:string, userlName:string, username:string, userPassword:string, 
        userEmail:string, userContactNo:string, userDOB:Date){

        this.userId=userId;
        this.userfName=userfName;	
        this.userlName=userlName;
        this.username=username;
        this.userPassword=userPassword;
        this.userEmail=userEmail;
        this.userContactNo=userContactNo;
        this.userDOB=userDOB;
    }

}

File: Backend/src/main/java/com/boot/entity/Userreg.java
package com.boot.entity;

import java.sql.Date;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Userreg {
	@Id
	@GeneratedValue
	@Column(name = "userId")
	private int userId;
	
	@Column(name = "userfName")
	private String userfName;
	
	@Column(name = "userlName")	
	private String userlName;
	
	@Column(name = "username")
	private String username;
	
	@Column(name = "userPassword")
	private String userPassword;
	
	@Column(name = "userEmail")
	private String userEmail;
	
	@Column(name = "userContactNo")
	private double userContactNo;
	
	@Column(name = "userDOB")
	private Date userDOB;
	
}
